src/js/samples/components/Create/CreateSample.tsx

```diff
@@ -126,50 +129,68 @@ type formValues = {
     sidebar: { labels: number[]; subtractionIds: string[] };
 };
 
-const getInitialValues = forceGroupChoice => ({
-    name: "",
-    isolate: "",
-    host: "",
-    locale: "",
-    libraryType: "normal",
-    readFiles: [],
-    group: forceGroupChoice ? "none" : null,
-    sidebar: { labels: [], subtractionIds: [] },
-});
-
-export function CreateSample({
-    error,
-    forceGroupChoice,
-    groups,
-    subtractions,
-    allLabels,
-    onLoadSubtractionsAndFiles,
-    onCreate,
-    onClearError,
-    onListLabels,
-}) {
-    useEffect(() => {
-        onLoadSubtractionsAndFiles();
-        onListLabels();
-    }, []);
-
-    const { data: readsResponse, isLoading: isReadsLoading }: { data: UnpaginatedFileResponse; isLoading: boolean } =
-        useListFiles(FileType.reads, false);
+function getInitialValues(forceGroupChoice) {
+    return {
+        name: "",
+        isolate: "",
+        host: "",
+        locale: "",
+        libraryType: "normal",
+        readFiles: [],
+        group: forceGroupChoice ? "none" : null,
+        sidebar: { labels: [], subtractionIds: [] },
+    };
+}
 
-    if (subtractions === null || allLabels === null || isReadsLoading) {
+export default function CreateSample() {
+    const { data: allLabels, isLoading: labelsLoading } = useFetchLabels();
```

Comment:
Nitpick: keeps the `is` for booleans convention and sounds more natural imo
```suggestion
    const { data: allLabels, isLoading: isLoadingLabels } = useFetchLabels();
```